staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -810,6 +810,10 @@ message ImageSpec {
     // If the runtime handler is unknown, the request should be rejected.
     // An empty string would select the default runtime handler.
     string runtime_handler = 19;
+    // Mount indicates that the OCI object (image or artifact) should be mounted.
+    bool mount = 20;
```

Comment:
?

Reply:
We need an indicator on image pull for the runtime to let them handle the pull as OCI mount: https://github.com/kubernetes/kubernetes/pull/125663/commits/e457925b388b9b2d4d96124f5d99ee0009deed2f#diff-b22ab169cdc70b2e968bb653f29cf0a1384d3bccaebeef96aebf2a5a40821bbeR150

Reply:
kk, not sure we would use that in the containerd case. In the crio case if the image is already locally present but was not pulled with mount = true, does the image have to be pulled again with mount = true?

Reply:
Yes, we pull again and the runtime decides if it just remounts or does a completely new pull. Mostly just a remount

Reply:
kk should / will / may... on pod create container requests that include a mount source configured to use the returned mount_ref... (hostpath/imageref)

not sure if you want to explain that here in the comment?

Reply:
I can add more details in a comment, yes.